auth/services/selfsigned/store.go

```diff
@@ -65,3 +74,30 @@ func (s *memorySessionStore) Delete(sessionID string) {
 	defer s.lock.Unlock()
 	delete(s.sessions, sessionID)
 }
+
+func (s *memorySessionStore) Start(ctx context.Context) {
```

Comment:
theoretically you'd need a `Stop()` function which is called on shutdown and waits until the goroutine has finished. Otherwise you could have runaway goroutines or corrupt state (although there's no persistent state here).

Reply:
Yes, and then the Stop function should wait until the go routine being stopped has actually finished. But there no resources to corrupt (by early termination) here